core/src/main/java/org/apache/brooklyn/core/objs/AbstractBrooklynObject.java

```diff
@@ -55,8 +57,18 @@
     @SetFromFlag(value = "tags")
     private final Set<Object> tags = Sets.newLinkedHashSet();
 
+    private RelationSupportInternal<PublicSelfType> relations = new ByObjectBasicRelationSupport<PublicSelfType>(getPublicThis(), new RelationChangedCallback());
```

Comment:
Obviously tests pass, but can't see where it gets initialized, constructor is not called during rebind.